bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/graphics/FontMetrics.java

```diff
@@ -194,6 +200,7 @@ public int hashCode() {
 }
 
 /**
+ * * @deprecated Use {@link #win32_new(TEXTMETRIC, int)} instead.
```

Comment:
Put it at the end and remove the extra `*`

Reply:
There is no need to deprecate methods marked with noreference as it's platform specific and thus not part of SWT API, feel free to remove it directly.

Reply:
Done